1_prep/src/munge_netCDFs.R

```diff
@@ -1,29 +1,155 @@
-#' @title Split each GCM netCDF into feather files specific to cells and time periods
-#' @decription Create a feather file with driver data for each gcm, 
-#' for each cell, for each time period
-#' @param gcm_nc filenames of GCM netCDF files
+#' @title Add burn-in and burn-out to raw meteo data for each cell
+#' @description function to add burn-in and burn-out to the raw meteo
+#' data by mirroring the data. From Jordan's GLM projection code: 
+#' https://github.com/jread-usgs/lake-temperature-process-models/blob/master/3_run/src/run_glm_utils.R#L62-L76
+#' @param meteo_data the raw meteo data for the cell, read in from 
+#' the netCDF file
+#' @param burn_in length of burn-in period, in days
+#' @param burn_out length of burn-out period, in days
+#' @return a munged dataframe of meteorological data that includes
+#' burn-in and burn-out time
+add_burn_in_out_to_meteo <- function(meteo_data, burn_in = 300, burn_out = 190){
```

Comment:
Why do we have this & `munge_gcm_dates()`. That second function makes some of these decisions already, so it seems like the output should be passed in here and used to actually add the burn in/out data.

Reply:
Okay, made the switch. I think it's all cleaner now.